docs/web/community/ecosystem.md

```diff
@@ -0,0 +1,21 @@
+# Ecosystem
+
+## Backends
+
+### [pandas](https://github.com/pandas-dev/pandas)
+
+[pandas](https://github.com/pandas-dev/pandas) is a Python package that provides fast, flexible, and expressive data structures designed to make working with "relational" or "labeled" data both easy and intuitive. It aims to be the fundamental high-level building block for doing practical, real world data analysis in Python. Additionally, it has the broader goal of becoming the most powerful and flexible open source data analysis / manipulation tool available in any language. It is already well on its way towards this goal.
```

Comment:
```suggestion
[pandas](https://pandas.pydata.org) is a Python package that provides fast, flexible, and expressive data structures designed to make working with "relational" or "labeled" data both easy and intuitive. It aims to be the fundamental high-level building block for doing practical, real world data analysis in Python. Additionally, it has the broader goal of becoming the most powerful and flexible open source data analysis / manipulation tool available in any language. It is already well on its way towards this goal.
```
Can you also add line breaks so lines are not more than 100 columns or so please? Easier to review, and to read in wide screens.

Reply:
Yes I'll definitely do that

Reply:
@datapythonista Do you think I should link to the pandas website instead of the GitHub repo?